File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/SortPagesMenuManager.java
package com.shevelev.comics_viewer.activities.comics_creation;

import android.view.Menu;

import com.shevelev.comics_viewer.common.OptionsMenuManager;

public class SortPagesMenuManager extends OptionsMenuManager
{
    public SortPagesMenuManager(Menu menu)
    {
        super(menu);
    }

    public void setAcceptVisible(boolean visible)
    {
        setVisible(0, visible);
    }
}

File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/SortPagesMenuManager.kt
package com.shevelev.comics_viewer.activities.comics_creation

import android.view.Menu
import com.shevelev.comics_viewer.common.OptionsMenuManager

class SortPagesMenuManager(menu: Menu?) : OptionsMenuManager(menu) {
    fun setAcceptVisible(visible: Boolean) {
        setVisible(0, visible)
    }
}